metal feature fails to compile

## Describe the bug
when trying to compile examples of mistralrs using metal features
```
cargo b -p mistralrs --examples --features metal
```

i'm getting:
```
 Compiling mistralrs-quant v0.7.0 (/Users/wiking/mistral.rs/mistralrs-quant)
error[E0599]: no method named `command_buffer` found for reference `&MetalDevice` in the current scope
    --> mistralrs-quant/src/utils/ops.rs:2309:33
     |
2309 |     let command_buffer = device.command_buffer()?;
     |                                 ^^^^^^^^^^^^^^ method not found in `&MetalDevice`

error[E0308]: mismatched types
    --> mistralrs-quant/src/utils/ops.rs:2317:9
     |
2314 |     crate::metal_kernels::call_fused_glu(
     |     ------------------------------------ arguments to this function are incorrect
...
2317 |         device.kernels(),
     |         ^^^^^^^^^^^^^^^^ expected `metal_kernels::Kernels`, found `candle_metal_kernels::Kernels`
     |
     = note: `candle_metal_kernels::Kernels` and `metal_kernels::Kernels` have similar names, but are actually distinct types
note: `candle_metal_kernels::Kernels` is defined in crate `candle_metal_kernels`
    --> /Users/wiking/.cargo/git/checkouts/candle-5b4d092929d18d36/8839457/candle-metal-kernels/src/kernel.rs:65:1
     |
  65 | pub struct Kernels {
     | ^^^^^^^^^^^^^^^^^^
note: `metal_kernels::Kernels` is defined in the current crate
    --> mistralrs-quant/src/metal_kernels/mod.rs:52:1
     |
  52 | pub struct Kernels {
     | ^^^^^^^^^^^^^^^^^^
note: function defined here
    --> mistralrs-quant/src/metal_kernels/mod.rs:2607:8
     |
2607 | pub fn call_fused_glu(
     |        ^^^^^^^^^^^^^^
...
2610 |     kernels: &Kernels,
     |     -----------------
```

## Latest commit or version
e2257d2170632674fd5f9ad33a9f9b9fb77f72be

--- a/mistralrs-quant/src/utils/ops.rs
+++ b/mistralrs-quant/src/utils/ops.rs
@@ -2306,15 +2306,15 @@ fn fused_glu_metal(a: &Tensor, b: &Tensor, activation: GluActivationType) -> Res
     };
 
     let device = a_metal.device();
-    let command_buffer = device.command_buffer()?;
-    command_buffer.set_label("fused-glu");
+    let encoder = device.command_encoder()?;
+    encoder.set_label("fused-glu");
 
     let output = device.new_buffer(n_elements, dtype, "fused-glu-output")?;
 
     crate::metal_kernels::call_fused_glu(
         device.device(),
-        &command_buffer,
-        device.kernels(),
+        &encoder,
+        &crate::metal_kernels::Kernels::new(),
         dtype,
         a_metal.buffer(),
         b_metal.buffer(),

